nidirect_landing_pages/nidirect_landing_pages.module

```diff
@@ -182,7 +182,7 @@ function nidirect_landing_pages_form_alter(&$form, FormStateInterface $form_stat
       $msg = "Note that the theme/subtheme that you choose here will be overridden by this landing page. <br/>";
       $msg .= "As an example, if you select 'Motoring / Road Safety' here then when you visit the page at ";
       $msg .= "/information-and-services/motoring there will be a link to this page labelled 'Road Safety'.";
-      $form['field_subtheme']['widget']['#description'] = t($msg);
+      $form['field_subtheme']['widget']['#description'] = $msg;
```

Comment:
Did `t()` not work for the description text here?

Reply:
Code standards will whine about it.